config.yaml

```diff
@@ -18,6 +18,9 @@ options:
     type: string
     default: 192.168.250.1
     description: Gateway IP address to the Core Network.
+  core-interface-mtu-size:
+    type: int
+    description: MTU size of UPF's Core Interface which is allowed between 0 < MTU <= 9000.
```

Comment:
Since we don't have a default value here in the config.yaml, we should tell users what will happen if they don't enter anything.

Reply:
And should we have a more reasonable minimum than 0? Do we envision any use case that would need anyting under 1400?

Reply:
We should also specify the unit (bytes)

Reply:
vxlan underlay causes main ifaces to be 1442, so adding multus calico ifaces over that results in MTU being 1372.  I'd say the lowest should be around 1200 to be safe.

There is also work being done into "super" jumbo frames, so 9000 isn't even a hard upper limit yet: https://en.wikipedia.org/wiki/Jumbo_frame#Super_jumbo_frames

I would suggest upper limit, if we even want one, should be at leat 64k

Reply:
Thinking more about this: other than it being an int, I don't think we should place any restriction on the value.  People won't be setting it if they don't know what they are doing, and there will always be an exception to whatever rule we come up with

Reply:
Done